specification/synapse/data-plane/Microsoft.Synapse/preview/2021-06-01-preview/entityTypes/DataFlow.json

```diff
@@ -128,6 +128,14 @@
           "description": "Transformation description.",
           "type": "string"
         },
+        "dataset": {
+          "description": "Dataset reference.",
+          "$ref": "../artifacts.json#/definitions/DatasetReference"
+        },
+        "linkedService": {
+          "description": "Linked service reference.",
+          "$ref": "../artifacts.json#/definitions/LinkedServiceReference"
+        },
```

Comment:
Adding some changes missed from the last commit here: https://github.com/Azure/azure-rest-api-specs/pull/16419